recipes/openmp/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "openmp" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "db55d85a7bb289804dc42fc5c8e35ca24dfc3885782261b675a194fd7e206e26" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://releases.llvm.org/{{ version }}/openmp-{{ version }}.src.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9   # [win and py27]
+    - vc14  # [win and py35]
+    - vc14  # [win and py36]
+  skip: true  # [win and py35]
+
+requirements:
+  build:
+    - toolchain
+    - llvmdev =={{ version }}
+    - cmake
+    - perl  ==5.22.2.1
+    - python  # [win]
+  run:
+    - clangdev =={{ version }}
```

Comment:
So there is some discussion about dropping this in `libcxx`. We will probably want to do the same thing here if they drop `libcxx`.